packages/react/src/useGlobalAction.ts

```diff
@@ -53,6 +55,7 @@ const processResult = (result: UseMutationState<any, any>, action: GlobalActionF
     data = get(result.data, [action.operationName]);
     if (data) {
       const errors = data.errors;
+      data = data.result;
```

Comment:
Is this change related to keeping the same identity?

Reply:
You've got laser eyes! Nope, this was extra, I noticed that we didn't actually return the right value for global actions at all. We returned the whole data object , like `{success: true, errors: [...], result: <something>}` instead of just the `<something>`. I was gonna land this as semver major for this reason, tho I doubt that many people are using `useGlobalAction` with custom results from the backend.